packages/data-layer/src/data-layer.ts

```diff
@@ -397,6 +402,84 @@ export class DataLayer {
     return response.rounds;
   }
 
+  async getRoundByIdAndChainIdWithApprovedApplications({
+    roundId,
+    chainId,
+  }: {
+    roundId: string;
+    chainId: number;
+  }): Promise<{ round: Round } | null> {
```

Comment:
@gravityblast I think it would make more sense to always return an object but set round to null, this way it can more backwards compatible as the type returned is always an object, what do you think?